artifactory/utils/npm/config-set.go

```diff
@@ -0,0 +1,24 @@
+package npm
+
+import (
+	gofrogcmd "github.com/jfrog/gofrog/io"
+	"github.com/jfrog/jfrog-client-go/utils/errorutils"
+)
+
```

Comment:
Let'e rena,e this file from `config-set.go` to `config_set.go`, so that ti is more aligned with go's naming conventions for files.